cmd/mount.go

```diff
@@ -392,6 +401,110 @@ func NewReloadableStorage(format *meta.Format, reload func() (*meta.Format, erro
 	return holder, nil
 }
 
+func tellFstabOptions(c *cli.Context) string {
+	opts := []string{"_netdev"}
+	for _, s := range os.Args[2:] {
```

Comment:
The `-d` argument cannot be used in fstab, otherwise it will fail to mount automatically.

Reply:
`-d` is already assumed in https://github.com/juicedata/juicefs/blob/main/cmd/main.go#L93, but surprisingly, the `--background` option will cause mount to crash, because `background` is classified as a FUSE option. I think this is a bug and I've changed it in [758b57c](https://github.com/juicedata/juicefs/pull/2462/commits/758b57cff1dce34b840dd4b1b1925e4907144526)